src/tests/ftest/util/ior_test_base.py

```diff
@@ -157,8 +159,19 @@ def run_ior_with_pool(self):
             self.ior_cmd.test_file.update(self.dfuse.mount_dir.value
                                           + "/testfile")
 
-        # Run IOR
-        self.run_ior(self.get_job_manager_command(), self.processes)
+        out = self.run_ior(self.get_job_manager_command(), self.processes,
+                           intercept)
+
+        if self.dfuse:
+            self.dfuse = None
+        if self.pool:        
+            errors = self.destroy_pools(self.pool)
+            if errors:
+                self.log.error("Error destroying pool. \n {}".format(
```

Comment:
(pylint-logging-format-interpolation) Use % formatting in logging functions and pass the % parameters as arguments